fuzz/ass.dict

```diff
@@ -0,0 +1,192 @@
+# SPDX-License-Identifier: CC0-1.0
+#
+# Keywords for dictionary-assisted ASS fuzzing
+# For use with AFL++'s -x and libfuzzer's -dict option
+#
+# Syntax:
+#   Lines starting with # are ignored.
+#   Keywords are listed as name="keyword", with the alphanumeric
+#   identifier name follooed by = being optional.
+#   Inside the quotes " and \ must be escaped by \" and \\ respectively
+#   and non-printable characters must be escaped as \x<hex>.
+
+# Some common font names
+"Arial"
+"DejaVu Sans"
+
+# Various common terminals
+"yes"
+"no"
+"0x"
+"&H"
+";"
+","
+"\x0a"
+"\x0d\x0a"
+"+"
+"-"
+":"
+
+# Sections
+"[Script Info]"
+"[Fonts]"
+"[V4 Styles]"
+"[V4+ Styles]"
+"[V4++ Styles]"
+"[Events]"
+
+# Script Info Headers
+"ScriptType"
+"PlayResX"
+"PlayResY"
+"Timer"
+"WrapStyle"
+"ScaledBorderAndShadow"
+"YCbCr Matrix"
+"Kerning"
+"Language"
+#"; Script generated by FFmpeg/Lavc"
```

Comment:
Is this commented out intentionally?

Reply:
Kind of, AFL++ was warning that such a large single dictionary entry may negatively affect fuzzing performance. Considering one sample with the comment will suffice for coverage and the effects can be acheived otherwise the fuzzer probably doesn't even need to be told about this being an active sequence. It would have been better to just remove it, but I didn't because I missed it before opening the PR — now I removed it properly.